data/tribes/initialization/atlanteans/units.lua

```diff
@@ -1450,21 +1464,30 @@ descriptions:new_tribe {
                pgettext("atlanteans_building", "The fisher’s house needs water full of fish within the work area."),
                -- TRANSLATORS: Note helptext for an atlantean production site: Fisher's House, part 2
                pgettext("atlanteans_building", "Build a fish breeder’s house close to the fisher’s house to make sure that you don’t run out of fish.")
-            }
+            },
+            -- TRANSLATORS: Note lore for an atlantean production site: Fisher's House use some local fise song as you like
+            lore = pgettext("atlanteans_building", [[‘Take your net and come to the sea<br>]] ..
+                                          [[In the earyly moning the moon ya still see<br>]] ..
+                                          [[Collect ya catch of fish for us and for you<br>]] ..
+                                          [[And Satul will warm the people of from your do.']]),
+            -- TRANSLATORS: Note lore_autor for an atlantean production site: Fisher's House
+            lore_author = pgettext("atlanteans_building", "A Song from the fishers guild")
          }
       },
       {
          name = "atlanteans_fishbreeders_house",
          helptexts = {
+            -- TRANSLATORS: Purpose helptext for an atlantean production site: Fish Breeder's House
+            purpose = pgettext("building", "Breeds fish."),
+            -- TRANSLATORS: Note helptext for an atlantean production site: Fish Breeder's House
+            note = pgettext("atlanteans_building", "The fish breeder needs open access to the coast."),
             -- TRANSLATORS: Lore helptext for an atlantean production site: Fish Breeder's House
             lore = pgettext("atlanteans_building", [[‘Only after the last duck has been shot down<br>]] ..
                                           [[Only after the last deer has been put to death<br>]] ..
                                           [[Only after the last fish has been caught<br>]] ..
                                           [[Then will you find that spiders are not to be eaten.’]]),
             -- TRANSLATORS: Lore author helptext for an atlantean production site: Fish Breeder's House
-            lore_author = pgettext("atlanteans_building", "Prophecy of the fish breeders"),
-            -- TRANSLATORS: Purpose helptext for an atlantean production site: Fish Breeder's House
-            purpose = pgettext("building", "Breeds fish.")
+            lore_author = pgettext("atlanteans_building", "Prophecy of the fishers guild")
```

Comment:
```suggestion
            lore_author = pgettext("atlanteans_building", "Prophecy of the fishers’ guild")
```

Reply:
I would stay with the original author here